src/nbl/video/IOpenGL_LogicalDevice.h

```diff
@@ -792,17 +788,17 @@ class IOpenGL_LogicalDevice : public ILogicalDevice, protected impl::IOpenGL_Log
                 _waitAll = true;
                 timeout = 0xdeadbeefBADC0FFEull;
```

Comment:
hmm, so in case _waitAll==false and _count!=1u, timeout is always 0 at first run? i dont get this

Reply:
or is it on purpose?

Reply:
if you're not waiting for all fences, it means that if any fence succeeds the function should return a sucess (according to the vulkan spec)

so even if the user passes in a large timeout, you cannot start waiting on the set of _count>=2 fences with large timeouts, first you need to poll them all (because a success from one should cause the execution to resume immediately)

Reply:
neat, ok, makes perfect sense